components/MarkdownEditor.tsx

```diff
@@ -120,6 +113,9 @@ export default function MarkdownEditor(props: {
               background-color: rgb(247, 247, 247);
               color: #3d3d3d;
             }
+            .w-md-editor-toolbar li > button {
+              height: 24px;
+            }
```

Comment:
Since there are built in buttons for the MDEditor which we might add at a later point I think we should select the specific button we are using now. This should work:

```CSS
.w-md-editor-toolbar button[data-name="link"] {
    height: 24px;
}
```

Reply:
Fixed